Consistence/Sniffs/Exceptions/ExceptionNameEndsWithExceptionSniff.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+declare(strict_types = 1);
+
+namespace Consistence\Sniffs\Exceptions;
+
+use PHP_CodeSniffer_File;
+use SlevomatCodingStandard\Helpers\ClassHelper;
+use StringHelper;
```

Comment:
I believe this is an incorrect `use`, this will load `StringHelper` from `Phing`, which is only available in dev, so it will fail when installed without it. -> `SlevomatCodingStandard\Helpers\StringHelper`